.github/workflows/libc-fullbuild-tests.yml

```diff
@@ -0,0 +1,76 @@
+# This workflow is for pre-commit testing of the LLVM-libc project.
+name: LLVM-libc Pre-commit Fullbuild Tests
+
+on:
+  pull_request:
+    branches: [ "main" ]
+    paths:
+      - 'libc/**'
+      - '.github/workflows/libc-fullbuild-tests.yml'
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - c_compiler: clang
+            cpp_compiler: clang++
+          # TODO: add back gcc build when it is fixed
+          # - c_compiler: gcc
+          #   cpp_compiler: g++
```

Comment:
Do you have more context about what's going wrong with the gcc builds?

Reply:
Please see https://github.com/llvm/llvm-project/actions/runs/12217700280/job/34082285018?pr=119104 and https://github.com/llvm/llvm-project/actions/runs/12218362957/job/34083763790?pr=119104